src/liballoc/collections/btree/node.rs

```diff
@@ -384,6 +379,7 @@ impl<BorrowType, K, V, Type> NodeRef<BorrowType, K, V, Type> {
     }
 
     pub fn keys(&self) -> &[K] {
```

Comment:
Same, should probably also get a doc comment.

Reply:
This one is public, so I agree entirely.